embedded/sql/sql_grammar.y

```diff
@@ -230,6 +230,11 @@ ddlstmt:
     {
         $$ = &DropIndexStmt{table: $4, cols: $6}
     }
+|
+    DROP INDEX IDENTIFIER DOT ids
```

Comment:
what about restricting to a single column with this syntax? 

DROP INDEX IDENTIFIER DOT IDENTIFIER